src/components/LoadingComments/LoadingComments.stories.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { LoadingComments } from "./LoadingComments";
+
+export default { component: LoadingComments, title: "LoadingComments" };
+
+export const Default = () => {
+  return (
+export const Default = () => <LoadingComments />
+      <LoadingComments />
```

Comment:
hehe you might need to remove this and the closing fragments as well :P  sorry

Reply:
No worries, done